.github/workflows/label-new-issues.yaml

```diff
@@ -0,0 +1,19 @@
+# This workflow is used to add the 'triage-needed' label to newly opened issues.
```

Comment:
should this file be moved to the scripts folder?

Reply:
No github workflows have to be in this specific directory. We can't do anything about it.